csm/core/data/models/rgw.py

```diff
@@ -31,7 +31,7 @@ class RgwErrors(Enum):
     InternalError = "Internal server error."
     NoSuchUser = "User does not exist."
     NoSuchBucket = "Bucket does not exist."
-    NoSuchKey = "No such access key."
+    NoSuchKey = "No such key."
```

Comment:
in which scenario, s3 server returns - NoSuchKey ?

Reply:
actually NoSuchKey is returned from bucket link operation.. if we give incorrect bucket name.. this issue pointed out by Amit

Reply:
got it.